src/Augmint.ts

```diff
@@ -152,7 +146,6 @@ export class Augmint {
             decimalsDiv: this.token.decimalsDiv,
```

Comment:
if we add the type for the var it then it won't happen in the future:
```
const options:IExchangeOptions = { ...
```

Reply:
Thought TS would be smart enough to infer the type. Fixed.